packages/api/src/models/reservation.schema.ts

```diff
@@ -10,7 +10,7 @@ export interface Reservation {
   to: Date;
   status: "pending" | "active" | "cancelled" | "archived";
```

Comment:
```suggestion
  status: ReservationStatus;
```

Reply:
Can you please add this import to this file:

```
import { ReservationStatus } from "./reservation-status";
```

Reply:
Yes!